documentation/steps-after-generations.md

```diff
@@ -12,6 +12,22 @@ rush update
 rush build -t <your-package-name>
 ```
 
+# Generate CHANGELOG.md
+
+Install `js-sdk-release-tools`
+```
+npm install -g @azure-tools/js-sdk-release-tools
+```
+
+After you build your package, run
+```
+changelog-tool <your-package-path>
+```
+Here is the example
+```
+changelog-tool sdk/advisor/arm-advisor
+```
+
 # Improve README.md document
```

Comment:
At the end of this document, the APIView section should be deleted as it is not telling people what to do with it.
Also for the `Release` section, update it to match the content on Step 6 of PR: https://github.com/Azure/azure-sdk-for-go/pull/23837/files